WikiDBUpdaterHttp/WikiDBUpdaterHttp.cs

```diff
@@ -0,0 +1,144 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using System.Linq;
+using Microsoft.Extensions.Configuration;
+using System.Collections.Generic;
+using System.Net.Http.Headers;
+using System.Net.Http;
+using Azure.Data.Tables;
+using WikiSentiment;
+
+
+namespace WikiDBUpdaterHttp
+{
+    /// <summary>
+    /// HTTP triggered function that updates DB with new wiki collections
+    /// </summary>
+    public class WikiDBUpdaterHttp
+    {
+        private ConfigurationWrapper config;
+
+        private string[] allLanguageCodes;
+
+        private HttpClient httpClient;
+
+        private Dictionary<string, string[]> articleExceptions;
+
+        public WikiDBUpdaterHttp(IConfiguration iConfig)
+        {
+            config = new ConfigurationWrapper(iConfig);
+
+            articleExceptions = config.GetValue<Dictionary<string, string[]>>
+                ("FunctionValues:CountryExceptions");
+
+            allLanguageCodes = config.GetValue<string[]>("FunctionValues:CountryCodes");
```

Comment:
```suggestion
            allLanguageCodes = config.GetValue<string[]>("FunctionValues:CountryCodes").ToHashSet();
```
See comment above about `HashSet`s